src/views/Profile/styles/Sidemenu.styles.ts

```diff
@@ -1,4 +1,5 @@
 import { makeStyles, Theme } from '@material-ui/core';
+import { callbackify } from 'util';
```

Comment:
Remove this unused import of 'callbackify'.

Reply:
Done